pkg/config/registry_config_test.go

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"testing"
+)
+
+func TestApplicationConfig(t *testing.T) {
+
+	err := Load(WithPath("./testdata/application.yaml"))
+	assert.Nil(t, err)
+
+	center := rootConfig.RegistryConf
+	serverConfig := rootConfig.ServerConf
+	assert.NotNil(t, center)
+	assert.NotNil(t, serverConfig)
+}
+
+func TestApplicationConfig1(t *testing.T) {
+
+	err := Load(WithPath("./testdata/registry.conf"))
+	assert.Nil(t, err)
+
+	center := rootConfig.RegistryConf
+	serverConfig := rootConfig.ServerConf
+	assert.NotNil(t, center)
+	assert.NotNil(t, serverConfig)
+}
+
+//结构体
+type stu struct {
+	id   string
+	name string
+	age  int
+}
+
+//方法
```

Comment:
Delete all Chinese characters

Reply:
In coding, I will pay attention to such issues in the future